proto/ibc/applications/transfer/v2/authz.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+
+package ibc.applications.transfer.v2;
```

Comment:
This proto file could live in `ibc.applications.transfer.v1` but its not required. 
The `v2` protos were added to support an updated `FungibleTokenPacketData`. Happy to leave it here.